test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py

```diff
@@ -64,6 +116,9 @@ def test_eks_tensorflow_albert(tensorflow_inference):
 
```

Comment:
nit: you can just add "neuron" to this condition and update the skip message to include neuron